nxc/protocols/winrm.py

```diff
@@ -334,3 +337,140 @@ def lsa(self):
             )
             LSA.dumpCachedHashes()
             LSA.dumpSecrets()
+
+    def dpapi(self):
+        """
+        Find and unlock Credential Manager masterkeys and credentials owned by user.
+        The flow is inspired by and a simplified version of dploot's triage methods for user masterkeys and credentials.
+        Actual decryption of keys and credentials is taken and adapted from impacket-dpapi.
+        """
+        user_masterkey_path = ntpath.join("C:\\Users", self.username, "AppData\\Roaming\\Microsoft\\Protect")
```

Comment:
will it be possible to check other user in case the user is admin ? therefore being able to also get other creds ?

Reply:
> PS: this is giving me interesting ideas for expanding the features, like searching for all reachable masterkey and creds files, attempting decryption with credentials we might have gathered in the nxc db, etc. But I think I would explore that in the context of follow-up PRs, making each of them easier to review and merge.

Agreed 👍